src/server/plugins/engine/components/DatePartsField.ts

```diff
@@ -151,8 +153,14 @@ export class DatePartsField extends FormComponent {
       }
     })
 
-    const firstError = errors?.errorList[0]
```

Comment:
This is the bug, it assumed the errors are for the current component only

(Errors should be filtered by name instead)